core/mondoo-slack-security.mql.yaml

```diff
@@ -0,0 +1,283 @@
+    docs:
+      desc: |
+        Having more than one administrator account is especially important to avoid single points of failure. For larger organizations, it is also useful to have several administrators to share the workload.
+      audit: |
+        Run the command in MQL and verify that the length is in the range between 2 and 4.
+
+          ```mql
+          slack.users.admins.length
+          ```
+      remediation: |
+        Make sure the number of Admin accounts is somewhere in the range of 2-4
+
+  - uid: mondoo-slack-security-admins-secure-2fa-methods
+    title: Ensure that admins use secure 2FA methods
+    mql: |
+      # needs resource
+      #slack.users.admins.all(has2FA == true && twoFactorType == "app")
+      if (false) {"check manually"}
+    impact: 70
+    docs:
+      desc: |
+        Administrator accounts should be secured with the more secure version of 2FA, which employs an application such as `Authy`, instead of relying on the less secure method via SMS, which is prone to SIM Swapping Attacks.
+      audit: |
+        Run the command in MQL and verify that 
+
+          ```mql
+          slack.users.admins {has2FA twoFactorType name}
+          ```
+      remediation: |
+        Make sure that Admin accounts only use the secure `app` setting and `sms` ist not allowed.
+
+  - uid: mondoo-slack-security-use-strong-factors
+    title: Ensure 2-Step Verification (Multi-Factor Authentication) is enforced for all users
+    mql: |
+      slack.users.members.all( has2FA == true || enterpriseUser != null || id=="USLACKBOT" )
+    impact: 60
+    docs:
+      desc: |
+        All user accounts should be protected with 2-factor authorization for enhanced protection against account takeovers by attackers.
+      audit: |
+        Run the command in MQL and verify that all users have 2-factor authorization enabled.
+
+          ```mql
+          slack.users.where(enterpriseUser == null || id !="USLACKBOT") {has2FA twoFactorType name}
+          ```
+      remediation: |
+        Make sure that all users have some form of 2FA enabled.
+
+  - uid: mondoo-slack-security-name-external-channels
+    title: Use clear naming for external channels
+    mql: |
+      slack.conversations.where(isExtShared && isChannel ).all(name == props.externalChannelName )
+    impact: 30
+    props:
+      - uid: externalChannelName
+        title: Enter your naming pattern for externally shared Slack channels
+        mql: |
+          return /ext|extern|ex_/
+    docs:
+      desc: |
+        A clear naming pattern for all channels externally shared with other organizations makes it easier to spot faulty channels, and can prevent the involuntary disclosure of internal information.
+      audit: |
+        Run the command in MQL and verify all externally shared channels adhere to your naming pattern.
+        Note: You need to specify your custom naming pattern in the properties of this policy `props.externalChannelName` .
+
+          ```mql
```

Comment:
I have the same question about whether this should be the full cnspec command. 😄